sos/report/plugins/aap_gateway.py

```diff
@@ -35,11 +35,9 @@ def setup(self):
             "/etc/ansible-automation-platform/gateway/*.cert",
         ])
 
-        self.add_cmd_output([
-            "aap-gateway-manage list_services",
-            "ls -ll /etc/ansible-automation-platform/",
-            "ls -ll /etc/ansible-automation-platform/gateway/",
-        ])
+        self.add_cmd_output("aap-gateway-manage list_services")
+        self.add_dir_listing("/etc/ansible-automation-platform/",
+                             recursive=True)
```

Comment:
We will additionally collect subdirs of `/etc/ansible-automation-platform/gateway/` but I doubt that will cause an issue (like, speculating, some fiel in nested dir has some secret name we should not collect - but I doubt that would ever happen). ACK